flopy/export/netcdf.py

```diff
@@ -152,6 +153,10 @@ def __init__(
         forgive=False,
         **kwargs,
     ):
+        if isinstance(output_filename, os.PathLike):
+            output_filename = str(
+                Path(output_filename).expanduser().absolute()
```

Comment:
Out of curiosity, why is `str` needed here?

Reply:
it's not strictly &mdash; previously `output_filename` was a `str`, switching to `Path` would require changes to L160 and L523 and I figured it may be safer to keep it a string in case any external usages had similar expectations